yal/tests/conftest.py

```diff
@@ -0,0 +1,9 @@
+from unittest import mock
+
+import pytest
+
+
+@pytest.fixture(autouse=True, scope="package")
+def mock_sleep():
+    with mock.patch("asyncio.sleep"):
+        yield
```

Comment:
This is to speed up tests, so that we don't need to wait for all the `sleep` calls